scripts/prepare-preview-builds.jq

```diff
@@ -6,4 +6,47 @@
 # of the version. Technically we'd want to bump the non-prerelease portion as
 # well if we wanted this to be SemVer-compliant, but it was simpler not to.
 # This is just for testing, it doesn't need to strictly follow SemVer.
-.version |= split("-")[0] + "-preview-\($hash)"
+.version |= split("-")[0] + "-preview.\($hash)" |
```

Comment:
[Updates from the Snaps monorepo](https://github.com/MetaMask/snaps/blob/main/scripts/prepare-preview-builds.jq#L9-L52) that also leverage workspace versions.